README.md

```diff
@@ -60,15 +60,15 @@ const config = {
 };
 ```
 
-| Name | Description | Kind |
-| --- | --- | --- |
-| build | Sets a build ID. (Default: `''`) | String |
-| tags | Tags to add to the uploaded Sauce job. (Default: `[]`) | String[] |
-| region | Sets the region. (Default: `us-west-1`) | `us-west-1` \| `eu-central-1` |
-| upload | Whether to upload report and assets to Sauce (Default: `true`) | boolean |
-| outputFile | The local path to write the sauce test report. | String |
-
-You can also use the `SAUCE_REPORT_OUTPUT_NAME` environment variable as an alternative to the `outputFile` reporter option in your playwright config.
+| Name           | Description                                                                                                                      | Kind                         |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------|------------------------------|
+| `build`        | Sets a build ID. <br> Default: `''`                                                                                              | `String`                     |
+| `tags`         | Tags to add to the uploaded Sauce job. <br> Default: `[]`                                                                        | `String[]`                   |
+| `region`       | Sets the region. <br> Default: `us-west-1`                                                                                       | `us-west-1` \| `eu-central-1`|
+| `upload`       | Whether to upload report and assets to Sauce <br> Default: `true`                                                                | `boolean`                    |
+| `outputFile`   | The local path to write the sauce test report. Can be set in env var `SAUCE_REPORT_OUTPUT_NAME`.                                 | `String`                     |
+| `syncAssetsDir`| Set the destination folder for syncing attachments. This can be configured via the `SAUCE_SYNC_ASSETS_DIR` environment variable. | `String`                     |
```

Comment:
The reporter shares a dual use right now. One primary use case is for the customer to report jobs to Sauce. The other, by our runners.

The `syncAssetsDir` is primarily for our own internal use by our runners, is it not? So pretend I'm a customer reading this doc here, what do I do with this field? What's the importance of it? I don't get it. Do I need this sync? It's confusing. I don't think it should be listed here.

If you still want it documented in the README, it perhaps needs a separate section.